types/npm.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://purl-spec.org/schemas/purl-type.schema-1.0.json",
+  "$id": "https://purl-spec.org/types/npm.json",
+  "type": "npm",
+  "name": "Node Package Manager (NPM)",
+  "description": "PURL type for Node.js packages managed by NPM.",
+  "definition": {
+    "namespace": {
+      "requirement": "optional",
+      "allowed_characters": "^[a-z0-9-]+$",
+      "case_rules": {
+        "sensitivity": "case-sensitive",
+        "normalization": "lowercase"
+      },
+      "native_label": "scope"
+    },
+    "name": {
+      "requirement": "required",
+      "allowed_characters": "^[a-z0-9-]+$",
```

Comment:
This is incorrect. NPM contains packages like [`pkg:npm/%2F18_wahajali/.adventure_game@4.0.2`](https://www.npmjs.com/package/@18_wahajali/.adventure_game/v/4.0.2).

Reply:
So, I am reading that NPM allows "scoped domains" (intended originally for local scope/ref. disambiguation) which companies and individuals are using for actual publishing packages... like your example "@18_wahajali/.adventure_game".  However, your link is further encoded (i.e., the '@' character).